v0.1/elements/electrodeContact.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/elements/electrodeContact.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SANDS definition of an electrode contact",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type", 
+    "@id",
+    "identifier",
+    "coordinate"
+    ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/ElectrodeContact"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier.",
+      "pattern": "sands/elements/electrodeContact/([A-Za-z0-9_-]+).json"
+    },
+    "identifier": {
+      "type": "string",
+      "description": "Identifier of the electrode contact within files."
+    },
+    "coordinate": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.schema.json#definitions/COORDINATE",
+      "description": "Central coordinate of the electrode contact."
+    },
+    "definedIn":{
+	  "type": "array",
+	  "minItems": 1,
```

Comment:
Same comment as for electrode (and the answer to my other question is this schema). Remove "minItems" and add it to the required properties.

Reply:
Here the conclusion is: If wiki is correct, and it should be required: Remove and add to required. If wiki is wrong: Fix wiki entry, remove anyway but don't add to required.

Reply:
see above